docs/guides/code_examples/creating_web_archive/manual_archiving_playwright_crawler.py

```diff
@@ -63,7 +62,8 @@ async def main() -> None:
         async def archiving_hook(context: PlaywrightPreNavCrawlingContext) -> None:
             # Ensure that all responses with additional resources are archived
             context.page.on(
-                'response', partial(archive_response, logger=context.log, writer=writer)
+                'requestfinished',
+                partial(archive_response, logger=context.log, writer=writer),
```

Comment:
`on("requestfinished")` returns a `Request` instance ([docs](https://playwright.dev/python/docs/api/class-page#page-event-request-finished)), so an additional `.response()` call is needed.